xbmc/games/addons/GameClient.cpp

```diff
@@ -276,6 +277,9 @@ bool CGameClient::OpenFile(const CFileItem& file, IGameAudioCallback* audio, IGa
   if (!InitializeGameplay(file.GetPath(), audio, video))
     return false;
 
+  m_inGameSaves.reset(new CGameClientInGameSaves(this, &m_struct));
```

Comment:
See the comment above from from spiff, I would have done it if it was possible, but m_struct is only valid after CGameClient::Initialize.